frontend/components/PersistLogin.tsx

```diff
@@ -65,6 +67,17 @@ const PersistLogin: FC<PropsWithChildren<unknown>> = ({
     tokens.accessToken ? setLoading(false) : verifyRefreshToken();
   }, []);
 
+  useEffect(() => {
+    if (typeof window !== 'undefined') {
+      window.addEventListener('storage', (e) => {
+        if (e.key === 'refreshToken' && !e.newValue) {
+          // automatically logout (we only need to return to login because another browser window already did the rest)
+          router.push('/login');
+        }
+      });
```

Comment:
Might be needed to unsubcribe the event listener , can do this with an abort signal
I believe the following should work? I'm not 100% sure if this is needed though,
and if it is, if only unsubscribing on a logout is enough since we call persistlogin from quite a few pages.
```
      const controller = new AbortController;
      const signal = controller.signal;
      window.addEventListener('storage', (e) => {
        if (e.key === 'refreshToken' && !e.newValue) {
          // automatically logout (we only need to return to login because another browser window already did the rest)
          router.push('/login');
          controller.abort();
        }
      }, {signal: signal});
```